.github/workflows/ci.yml

```diff
@@ -87,10 +86,6 @@ jobs:
             command: bin/test.sh 'slow/testOnly -- tests.gradle.*'
             name: Gradle integration
             os: ubuntu-latest
-          - type: pants
```

Comment:
Don't we want to keep the tests, but just add a dependency on fastpass?

Reply:
The current Pants integration for "import build" doesn't work reliably. It's better for users to run the fastpass refresh/create commands from the command-line instead.